README.md

```diff
@@ -70,7 +70,7 @@ To do so, add the following to the `.pre-commit-config.yaml` file at the root of
 ```yaml
 repos:
   - repo: https://github.com/wemake-services/dotenv-linter
-    rev: master
+    rev: 0.2.0  # Use the ref you want to point at
```

Comment:
Let's use the real version: https://github.com/wemake-services/dotenv-linter/releases/tag/0.1.5

Reply:
Can do this change, but are you sure? The example config will not work then as is, because `0.1.5` was tagged before `.pre-commit-hooks.yml` file was added to this repo.

Reply:
Oh, my bad! 

I guess we have to release a new version then.